Vagrant-setup/travis_hadoop_files/core-site.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
```

Comment:
Did you manage to setup Hadoop in Travis ? Unless you setup, can we remove Hadoop-related config files ?